Unable to build - "Package 'soupsieve' requires a different Python"

As of today, the sync server seems to be impossible to build, as one of the current dependencies, Soupsieve, no longer supports Python 2. As per the build log:

```
Collecting soupsieve>=1.2
  Using cached https://www.piwheels.org/simple/soupsieve/soupsieve-2.0-py2.py3-none-any.whl (35 kB)
ERROR: Package 'soupsieve' requires a different Python: 2.7.16 not in '>=3.5'
make: *** [Makefile:29: local/COMPLETE] Error 1
```

Is there a way to bypass this in order to use a Python 2 - compatible version of Soupsieve, or will I have to wait until the Rust rewrite is ready for production?

--- a/requirements.txt
+++ b/requirements.txt
@@ -9,5 +9,6 @@ zope.component==4.2.1
 configparser==3.5
 mozsvc==0.9
 futures==3.0
+soupsieve==1.9.5
 https://github.com/mozilla-services/tokenserver/archive/1.4.5.zip
 https://github.com/mozilla-services/server-syncstorage/archive/1.6.14.zip

